bin/reth/src/node/mod.rs

```diff
@@ -317,7 +317,7 @@ impl Command {
         config: &Config,
         db: Arc<Env<WriteMap>>,
         executor: TaskExecutor,
-    ) -> NetworkConfig<ShareableDatabase<Arc<Env<WriteMap>>>> {
+    ) -> NetworkConfig<Arc<ShareableDatabase<Arc<Env<WriteMap>>>>> {
```

Comment:
Do we need it to be `Arc<ShareableDatabase<...>>`? Is `ShareableDatabase` not `Clone` in itself?